storm-clojure/src/main/clj/org.apache.storm/clojure.clj

```diff
@@ -0,0 +1,201 @@
+;; Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The directory for this I think is wrong.  It should be `org/apache/storm/clojure.clj` I'm not really sure if it makes a difference in the build, but it is not standard with the other code.